### PR TITLE
fixed table header for transaction table

### LINKED ISSUE #1319: "Type" is wrapped when there is no content below
Screen Shot 2020-05-24 at 10 05 56 PM

Should look alike
Screen Shot 2020-05-24 at 10 05 46 PM

### PR DESCRIPTION
**Pull request by ketanb56up**
**What type of PR is this?**
> Uncomment only one ` /kind <>` line, hit enter to put that in a new line, and remove leading whitespaces from that line:
>
> /kind api-change
> /kind backend-change

kind ui-change
> /kind bug
> /kind cleanup
> /kind design
> /kind documentation
> /kind failing-test
> /kind feature
> /kind flake

**What this PR does / why we need it**:
Fixed table header 'Type' to show in one line when data is not present. 
**Which issue(s) this PR fixes**:
This PR fixes the table header when the data is not there in the table data. 

Fixes #1319 

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note

```

### CODE REVIEW

**Comment by ketanb56up** [comment]
The table header Type have fixed design when data is present and when data is not present.
Screenshot from 2020-05-25 18-40-14

**Comment by ketanb56up** [comment]
Screenshot from 2020-05-25 18-41-34
When data is present.

File: src/shared/pages/action-list-page.tsx [hunk 1]
```diff
@@ -100,7 +100,7 @@ const getActionListColumns = (): Array<ColumnProps<ActionInfo>> => [
   {
     title: t("action.type"),
     dataIndex: "name",
-    width: "3vw",
+    width: "8vw",
```
Comment (puncsky) [inline src/shared/pages/action-list-page.tsx:103]: please fix all the places that are hard-coding width to a magic value. we should probably not set up a fixed width

File: src/shared/pages/action-list-page.tsx [hunk 2]
```diff
@@ -207,6 +217,12 @@ export const ActionTable: React.FC<IActionTable> = ({
             message: `failed to query actions in ActionTable: ${error}`
           });
         }
+        const widths = {
```
Comment (hunglmtb) [inline src/shared/pages/action-list-page.tsx:220]: it is still hard value, let's refer antd design of table component to find how to set the width in better way

**Comment by ketanb56up** [comment]
Screenshot from 2020-07-16 17-51-40
Screenshot from 2020-07-16 17-51-52

**Comment by liaojikunwork** [comment]
Not necessary

**Pull request closed without merging**